src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ExceptionElasticSearchService.java

```diff
@@ -72,13 +76,13 @@ public ConnectionSummaryDto searchForPage(String searchQuery, Pageable pageable)
 
       LOG.debug("Query {}", fullQuery);
 
-      Page<ExceptionView> result = exceptionElasticSearchRepository.search(fullQuery, pageable);
+      Page<DiscrepanciesView> result = discrepanciesElasticSearchRepository.search(fullQuery, pageable);
```

Comment:
Once again, using the deprecated repository search method for now as it's convenient and would need a bigger rewrite, can make decisions about this when we implement filtering